ui/app/AppLayouts/Chat/popups/community/AccessExistingCommunityPopup.qml

```diff
@@ -19,7 +19,8 @@ StatusModal {
     property var store
 
     function validate(communityKey) {
-        return communityKey.trim() !== ""
+        const bytes32Len = 66;
+        return communityKey.length === bytes32Len && Utils.isHex(communityKey) && Utils.startsWith0x(communityKey)
```

Comment:
I think that checking length == 66 should be moved to Utils.qml. There are a lot of functions with such checks, eg. isPrivateKey(). We should keep these numbers (66, 64, 42) in one place. So I suggest adding a new function, eg. Utils.isCommunityKey().